pkg/syncer/fullsync.go

```diff
@@ -138,49 +151,57 @@ func CsiFullSync(ctx context.Context, metadataSyncer *metadataSyncInformer) erro
 	return nil
 }
 
-// fullSyncCreateVolumes create volumes with given array of createSpec
-// Before creating a volume, all current K8s volumes are retrieved
-// If the volume is successfully created, it is removed from cnsCreationMap
-func fullSyncCreateVolumes(ctx context.Context, createSpecArray []cnstypes.CnsVolumeCreateSpec, metadataSyncer *metadataSyncInformer, wg *sync.WaitGroup, migrationFeatureStateForFullSync bool) {
+// fullSyncCreateVolumes create volumes with given array of createSpec.
```

Comment:
Not your change.

Nit: `create` -> `creates`.

Reply:
Done.